2025-01-20 20:53:05,527 INFO:  * Restarting with stat [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:53:05,969 INFO: Inicializace aplikace... [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:47]
2025-01-20 20:53:06,738 INFO: Aplikace byla úspěšně inicializována [in /Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/__init__.py:147]
2025-01-20 20:53:06,739 WARNING:  * Debugger is active! [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:53:06,745 INFO:  * Debugger PIN: 278-609-759 [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:53:06,753 INFO: 127.0.0.1 - - [20/Jan/2025 20:53:06] "GET /socket.io/?EIO=4&transport=polling&t=PI5HgBb HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:53:06,767 INFO: 127.0.0.1 - - [20/Jan/2025 20:53:06] "POST /socket.io/?EIO=4&transport=polling&t=PI5HgF7&sid=F_RzOh2FyGfKO049AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:53:06,771 INFO: 127.0.0.1 - - [20/Jan/2025 20:53:06] "GET /socket.io/?EIO=4&transport=polling&t=PI5HgFB&sid=F_RzOh2FyGfKO049AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:53:06,780 INFO: 127.0.0.1 - - [20/Jan/2025 20:53:06] "GET /socket.io/?EIO=4&transport=polling&t=PI5HgFN&sid=F_RzOh2FyGfKO049AAAA HTTP/1.1" 200 - [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
2025-01-20 20:53:23,969 INFO:  * Detected change in '/Users/martinmikes/Downloads/my-directus-project/root/supplo.ai/app/config/production.py', reloading [in /opt/anaconda3/lib/python3.11/site-packages/werkzeug/_internal.py:96]
